### PR TITLE
Update for OoT/MM NTSC 1.0 & DEBUG GC MQ PAL (Sealed Palace)

### PR DESCRIPTION
**Pull request by Anthrogi50**
Zelda 64.psm1: Changed the trail (inner) & (outer) to (start) & (end) respectively along with updating the description for the changes brought about listed later in the psm1 files for OoT & MM below.

Both NTSC 1.0: Added a deku stick range option and added the base start and end offset segments to the sword trail offset changes along with transparency options for deciding the how solid the sword trail should be during the start or end and on whichever half of the sword as well.

OoT NTSC 1.0: Fast Charge Lv2 readded after a fix. Some text revisions for 'hitbox' and so.

MM NTSC 1.0: Fast Magic Arrows added for fire, ice & light arrows. Some text revisions for 'equipment' and so. Also, fixed the offset order for the 'sword trail' colors. Removed erroneous offsets at the 'sword trail duration' check function. 

OoT (Sealed Palace) DEBUG GC MQ PAL: Fast Charge Lv2 readded after a fix. Added broken_knife, deku_stick & megaton_hammer range options along with magic spin colors and the starting & ending tip/base segments for the sword trail along with transparency options for deciding the how solid the sword trail should be during the start or end and on whichever half of the sword as well.

### CODE REVIEW

File: Files/Scripts/Options/Majora's Mask.psm1
Comment (Admentus64) [inline Files/Scripts/Options/Majora's Mask.psm1:516]: Do not change this

File: Files/Scripts/Options/Majora's Mask.psm1
Comment (Admentus64) [inline Files/Scripts/Options/Majora's Mask.psm1:2451]: Deku Stick change addition is good, but do not change anything else.

File: Files/Scripts/Options/Majora's Mask.psm1
Comment (Admentus64) [inline Files/Scripts/Options/Majora's Mask.psm1:2561]: Do not change this

File: Files/Scripts/Options/Ocarina of Time.psm1
Comment (Admentus64) [inline Files/Scripts/Options/Ocarina of Time.psm1:653]: Explain what BEFF80 and BEFF88 are needed for.

File: Files/Scripts/Options/Ocarina of Time.psm1
Comment (Admentus64) [inline Files/Scripts/Options/Ocarina of Time.psm1:2698]: Do not change this

File: Files/Scripts/Options/Ocarina of Time.psm1
Comment (Admentus64) [inline Files/Scripts/Options/Ocarina of Time.psm1:3429]: Not a fan of this, I would rather see this and it's code changes removed

File: Files/Scripts/Options/Ocarina of Time.psm1
Comment (Admentus64) [inline Files/Scripts/Options/Ocarina of Time.psm1:3542]: Deku Stick change addition is good, but do not change anything else.

File: Files/Scripts/Options/Ocarina of Time.psm1
Comment (Admentus64) [inline Files/Scripts/Options/Ocarina of Time.psm1:3949]: Do not change this

File: Files/Scripts/Options/The Sealed Palace.psm1
Comment (Admentus64) [inline Files/Scripts/Options/The Sealed Palace.psm1:111]: Do not change this, it's not related to your changes

File: Files/Scripts/Options/The Sealed Palace.psm1
Comment (Admentus64) [inline Files/Scripts/Options/The Sealed Palace.psm1:122]: Explain what 1B9A980 and 1B9A988 are needed for.

File: Files/Scripts/Options/The Sealed Palace.psm1
Comment (Admentus64) [inline Files/Scripts/Options/The Sealed Palace.psm1:167]: Limit your changes to just the DekuStick

File: Files/Scripts/Options/The Sealed Palace.psm1
Comment (Admentus64) [inline Files/Scripts/Options/The Sealed Palace.psm1:166]: You do not need to state you ported it, the code changes were yours to begin in the first place. Only do that when it isn't yours to begin with, then you ported it over.

File: Files/Scripts/Options/The Sealed Palace.psm1
Comment (Admentus64) [inline Files/Scripts/Options/The Sealed Palace.psm1:330]: Not a fan of this, I would rather see this and it's code changes removed

File: Files/Scripts/Options/The Sealed Palace.psm1
Comment (Admentus64) [inline Files/Scripts/Options/The Sealed Palace.psm1:307]: Deku Stick change addition is good, but do not change anything else.

File: Files/Scripts/Zelda 64.psm1
Comment (Admentus64) [inline Files/Scripts/Zelda 64.psm1:1275]: Do not change this, since I don't like the tip alpha option

**Review by Admentus64 — CHANGES_REQUESTED** [review]
Looks like it is on the right track.

Few things that have to have still though:
1. Limit your changes to what you want to change
2. Try not rename things if it's not related to your current active PR goal
3. Not a fan of both new alpha options. More on this below.
4. Explain changes to the offsets you made to existing options. What's the goal and reason here. Show screenshots for comparison.
5. Show screenshot comparisons for any visual/cosmetic/color changes you make

Once this is all addressed I start starting it for base OoT and base MM. And once those tests are passed I'll approve it.

About both new alpha options. I will test those, But first, show some comparison screenshots to make a case for it. Any cosmetic changes you have to compare through screenshots before we test it. The option may still be added if it offers enough of a visual change after I seen it ingame myself (screenshot comparison first here).

File: Files/Scripts/Options/Majora's Mask.psm1
Comment (Anthrogi50) [reply Files/Scripts/Options/Majora's Mask.psm1:516]: Oh I didn't do anything to this. Must be affected by line change.

File: Files/Scripts/Options/Majora's Mask.psm1
Comment (Anthrogi50) [reply Files/Scripts/Options/Majora's Mask.psm1:2417]: I will show this first.

File: Files/Scripts/Options/Majora's Mask.psm1
Comment (Anthrogi50) [reply Files/Scripts/Options/Majora's Mask.psm1:2561]: This too, I did not change.

File: Files/Scripts/Options/Majora's Mask.psm1
Comment (Anthrogi50) [reply Files/Scripts/Options/Majora's Mask.psm1:2451]: Thought it would be good to change the description a little because I found a more clearer sounding way to say the sentence that was originally "Set the length of the hitbox of the (sword name here)" because "of the" was said twice and while it's possible that could be needed and not changed, I found a better way to write that which was "Set the hitbox length of the (sword name here)" which sounded better for me and reduced the spaces a little but it's my opinion. If you don't want that, I accept your choice.

File: Files/Scripts/Options/Ocarina of Time.psm1
Comment (Anthrogi50) [reply Files/Scripts/Options/Ocarina of Time.psm1:653]: BEFF80 is the starting segment of the base color trail with BEFF88 being the ending segment version of the base color. I paired them with their respective segments of start to make their effects happen consistently with them, but this too I can show as well.

File: Files/Scripts/Options/Ocarina of Time.psm1
Comment (Anthrogi50) [reply Files/Scripts/Options/Ocarina of Time.psm1:2698]: Another space thing I assume.

File: Files/Scripts/Options/Ocarina of Time.psm1
Comment (Anthrogi50) [reply Files/Scripts/Options/Ocarina of Time.psm1:3429]: Will show this.

File: Files/Scripts/Options/Ocarina of Time.psm1
Comment (Anthrogi50) [reply Files/Scripts/Options/Ocarina of Time.psm1:3542]: Same reasons as before in a previous change within the MM.psm1

File: Files/Scripts/Options/Ocarina of Time.psm1
Comment (Anthrogi50) [reply Files/Scripts/Options/Ocarina of Time.psm1:3949]: Another space issue.

File: Files/Scripts/Options/The Sealed Palace.psm1
Comment (Anthrogi50) [reply Files/Scripts/Options/The Sealed Palace.psm1:111]: You have since told me you'll do the spacing, so I understand.

File: Files/Scripts/Options/The Sealed Palace.psm1
Comment (Anthrogi50) [reply Files/Scripts/Options/The Sealed Palace.psm1:122]: Same reasons as before in the ntsc 1.0 OoT.psm1

File: Files/Scripts/Options/The Sealed Palace.psm1
Comment (Anthrogi50) [reply Files/Scripts/Options/The Sealed Palace.psm1:167]: The broken knife and megaton hammer were missing, so I added those for 'sealed palace'.

File: Files/Scripts/Options/The Sealed Palace.psm1
Comment (Anthrogi50) [reply Files/Scripts/Options/The Sealed Palace.psm1:166]: I see although for me, I used the indication "ported" because even if I found the function, it's a copy over from the originally intended baserom I supported it with which was ntsc 1.0 for both OoT & MM, so I felt the need to but I'll let that go then.

File: Files/Scripts/Options/The Sealed Palace.psm1
Comment (Anthrogi50) [reply Files/Scripts/Options/The Sealed Palace.psm1:330]: Same reason previous, I will show it first.

File: Files/Scripts/Options/The Sealed Palace.psm1
Comment (Anthrogi50) [reply Files/Scripts/Options/The Sealed Palace.psm1:307]: Same reason with OoT.psm1

File: Files/Scripts/Zelda 64.psm1
Comment (Anthrogi50) [reply Files/Scripts/Zelda 64.psm1:1275]: This is actually referring to the correction of the sword trail **color** and not the transparency of the trail here actually. I will show this too.

**Review by Anthrogi50 — COMMENTED** [review]
I have left replies and will get to showing you some of the things you're uncertain about.

**Comment by Anthrogi50** [comment]
Alright, I turned the starting alphas of the sword trail into 3 choices and made rearrangements to the offset combined assignment to the tips and bases. That should be all.

**Comment by Anthrogi50** [comment]
Conflicts fixed, that should really be it now.

**Comment by Anthrogi50** [comment]
Currently waiting for you to reply now.

**Pull request closed without merging**
